lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/models/comment_info.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:analyzer/dart/ast/token.dart';
+
+import 'comment_type.dart';
+
+class CommentInfo {
```

Comment:
Please import all rule parts with `part of` in order to not make them public for other code